proposals/2019/Building Footprint.md

```diff
@@ -0,0 +1,17 @@
+# Add Footprint to Building Element
+
+## Overview
+
+The current structure is to have the footprint shape defined at the subsection level. The purpose of the proposal is to allow the footprint shape to also be defined at the building level.
+
+## Justification
+
+The footprint can be defined at both levels, that is, the building and subsection level. For example a building can have an overall rectangular footprint, but the underlying subsections can have L-shape footprints.
```

Comment:
This is simply not a compelling argument and is adding meaning to the building/subsection relationship that is not presently there and will call into question current example files. Geometry is an area of the schema that needs some work, and this will only exacerbate the problem. Finally, since there's no way to orient the footprints-within-footprints, I don't really see value in doing this a convenience when it is already possible.